sats-dna/src/main/kotlin/com/sats/dna/components/SatsFancyTopAppBar.kt

```diff
@@ -306,6 +325,17 @@ private fun HeaderImage(
                 contentScale = ContentScale.Crop,
             )
         }
+    }
+}
+
+@Composable
+private fun Header(
+    expandPercent: Float,
+    modifier: Modifier = Modifier,
+    content: @Composable () -> Unit,
+) {
+    Box(modifier) {
+        content()
 
         HeaderImageShade(expandPercent, Modifier.fillMaxSize())
     }
```

Comment:
I'm a little bit unsure of how to do this part. We'll want the image itself to have a shade, but probably not things like a paging indicator (Club Details image gallery) or a “new release” tag (GX Session/Type Details). :thinking:

Reply:
Solved by providing the shade through a parameter to `content`, much like `BasicTextField` does with its `innerTextField`.